cap_anndata/cap_anndata.py

```diff
@@ -91,31 +75,110 @@ def _read_df(self, h5_group: h5py.Group, columns: List[str]) -> CapAnnDataDF:
             cols_to_read = [c for c in columns if c in column_order]
             df = CapAnnDataDF()
             df.column_order = column_order
-
             index_col = self._read_attr(h5_group, "_index")
             df.index = read_elem(h5_group[index_col])
 
             for col in cols_to_read:
                 df[col] = read_elem(h5_group[col])
+
         if df.column_order.dtype != object:
             # empty DataFrame will have column_order as float64
             # which leads to failure in overwrite method
             df.column_order = df.column_order.astype(object)
         return df
 
-    @staticmethod
-    def _read_attr(obj: Union[h5py.Group, h5py.Dataset], attr_name: str) -> any:
-        attrs = dict(obj.attrs)
-        if attr_name not in attrs.keys():
-            raise KeyError(f"The {attr_name} doesn't exist!")
-        return attrs[attr_name]
+    def _write_elem_lzf(self, dest_key: str, elem: any) -> None:
+        write_elem(self._file, dest_key, elem, dataset_kwargs={"compression": "lzf"})
+
+
+class RawLayer(BaseLayerMatrixAndDf):
+    def __init__(self, h5_file: h5py.File):
+        super().__init__(h5_file, path_to_content="/raw/")
+        self._var: CapAnnDataDF = None
+
+    @property
+    def var(self) -> CapAnnDataDF:
+        if self._var is None:
+            self._var = self._lazy_df_load("var")
```

Comment:
We will need to move all strings like `var`, `obs`, `uns` to constants/enum, feel free to add refactoring issue